src/components/ButtonMenu/readme.md

```diff
@@ -454,3 +454,254 @@ class ButtonMenuExample extends React.Component {
 }
 <ButtonMenuExample/>
 ```
+
+##### ButtonMenu
+
+```js
+import React from 'react';
+import { ButtonMenu, MenuItem, Card } from 'react-rainbow-components';
+import { faEllipsisV } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import styled from 'styled-components';
+
+const Title = styled.h1.attrs(props => props.theme.rainbow)`
+    font-family: 'Lato Light';
+    font-size: 24px;
+    text-align: left;
+    color: ${props => props.palette.text.main};
+`;
+
+const CardSubtitle = styled.h4.attrs(props => props.theme.rainbow)`
+    font-family: Lato;
+    font-size: 12px;
+    color: ${props => props.palette.text.header};
+    display: block;
+`;
+
+const Label = styled.span.attrs(props => props.theme.rainbow)`
+    font-family: Lato Bold;
+    font-size: 24px;
+    line-height: 24px;
+    color: ${props => props.blue? props.palette.brand.main : props.palette.text.main};
+    text-transform: uppercase;
+`;
+
+const Description = styled.span.attrs(props => props.theme.rainbow)`
+    font-family: Lato;
+    font-size: 14px;
+    color: ${props => props.palette.text.disabled};
+`;
+
+const ColumnContainer = styled.div.attrs(props => props.theme.rainbow)`
+    min-height: 92px;
+    border-left: ${props => props.noBorder? "0":"1"}px solid ${props => props.palette.text.disabled};
```

Comment:
it is more readable if we use hasBorder and then if it is true then set the border-left and if not then do nothing like:
```
${props => props.hasBorder && `
    border-left: 1px solid ${props => props.palette.text.disabled};
`}
```